### PR TITLE
Adds medtek cartridges to Nano/Civimed

### PR DESCRIPTION
**Pull request by dustylens**
## About the PR

Adds medtek cartridges to the civi and nanomeds.
I also made the medical folders for the mini infinite because, well, why not? I mean legitimately why not.

## Why / Balance

Was overlooked when distributing cartridges.

## How to test

Look in the vends. Observe.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- add: Adds medtek cartridges to the medical vending machines.

### CODE REVIEW

**Review by dvir001 — APPROVED** [review]
Sure

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@
     BikeHornImplanterLabeled: 10 # Frontier
     LightImplanterLabeled: 10 # Frontier
     BoxFolderMedicalDoctor: 4294967295 # Frontier - infinite
+    MedTekCartridge: 4294967295 # Frontier - infinite
```
Comment (dvir001) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml:19]: 3 to 3 with HandheldHealthAnalyzer

Comment (ErhardSteinhauer) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml]: Two by two, hands of blue

**Comment by dvir001** [comment]
PR Looks good, can be merged

**Pull request merged**